File: src/test/java/com/hank/StudentTest.java
package com.hank;

import com.hank.student.Student;
import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class StudentTest {
    @Test
    public void highestScoreTest(){
        Student student = new Student("hank",60,80);
        Assertions.assertEquals(80,student.highest());
    }
    @Test
    public void averageTest(){
        Student student = new Student("Hank",60,80);
        Assertions.assertEquals((60+80)/2,student.getAverage());
    }
}

File: src/test/kotlin/com/hank/StudentTest.kt
package com.hank.kolin

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class StudentTest {
    @Test
    fun highestTest(){
        val stu = Student("Hank",66,80)
        Assertions.assertEquals(80,stu.highest())
    }
    @Test
    fun averageTest(){
        val stu = Student("Hank",66,80)
        Assertions.assertEquals((66+80)/2,stu.getAverage())
    }
    @Test
    fun gradingTest(){
        val stu = Student("Hank",66,80)
        Assertions.assertEquals('C',stu.grading())
    }

}